src/Jackalope/Transport/DoctrineDBAL/RepositorySchema.php

```diff
@@ -219,4 +224,35 @@ public function reset()
             $this->connection->exec($sql);
         }
     }
+
+    private function getMaxIndexLength($currentMaxLength = null)
+    {
+        if (-1 === $this->maxIndexLength) {
+            $this->maxIndexLength = null;
```

Comment:
@alexander-schranz i continued on your pull request. before, we did not pass a length - i feel its better to keep it that way by default. this code is not only for mysql but also postgres and sqlite, i am afraid of side effects if we suddenly limit to 255 characters.